source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -4513,34 +4526,57 @@ export class Media extends CardElement {
                 this._mediaElement.appendChild(src);
             });
             element.appendChild(this._mediaElement);
-
-            var play_button = this.makeButton("▶️");
-            play_button.onclick = e => this.play();
-            element.appendChild(play_button);
-
-            var pause_button = this.makeButton("⏸️");
-            pause_button.onclick = e => this.pause();
-            element.appendChild(pause_button);
-
-            var stop_button = this.makeButton("⏹️");
-            stop_button.onclick = e => this.stop();
-            element.appendChild(stop_button);
-        }
+			
+			let controlBar = <HTMLDivElement>document.createElement("div");
+			controlBar.className = this.hostConfig.makeCssClassName("ac-controlbar");
+			controlBar.style.whiteSpace = "nowrap";
+
+            this._playButton = Media.makeButton("▶️", "play", ()=>this.play());
```

Comment:
There seems to be a one character misalignment in the next few lines. VS Code has had that bug of introducing these for a while now.

Reply:
Let's start "play", "pause" and "reset" with capital letters given they will be displayed.

Reply:
"()=>this.play()" should be "() => this.play()"

Reply:
- Whitespaces, yes. Thanks for understanding. I will see to this.
- Will do the Play, Pause, Reset for title and lowercase for the CSS classes.
- Yes, thanks for noticing.

Reply:
by the way, how do you feel about having CSS classes like that? I normally like very much the fact that it blends well to selecting and changing stuff on any level of the stylesheet. On the other hand it can look too verbose. Would love your opinion.